snmp/datadog_checks/snmp/data/profiles/_generic-router-tcp.yaml

```diff
@@ -15,6 +15,7 @@ metrics:
     forced_type: monotonic_count
   - MIB: TCP-MIB
     symbol: tcpCurrEstab
+    forced_type: monotonic_count
```

Comment:
That's incorrect, this is a current number, not a monotonic count.

Reply:
Agreed, more info here: http://cric.grenoble.cnrs.fr/Administrateurs/Outils/MIBS/?oid=1.3.6.1.2.1.6.9

Nom | tcpCurrEstab
-- | --
Acces | readonly
Status | current
Description | The number of TCP connections for which the current state is either ESTABLISHED or CLOSE-WAIT.
Syntaxe | Gauge32 (SNMPv2-SMI)